rego-policy/deprek8.rego

```diff
@@ -0,0 +1,105 @@
+package main
```

Comment:
I think there are better k8s rego files that we can use than the one from deprek8. I'm thinking specifically about referencing the ones found here: https://github.com/swade1987/deprek8ion

That being said, I'm not sure carrying the policies themselves in the repo itself is necessary. I think as part of our workflow, we can have a step that retrieves our desired policies from wherever they happen to live. This way we're not having to keep these things in sync.